cypress/integration/team/teammates_pagination_spec.js

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/* eslint max-nested-callbacks: ["error", 7] */
+
+describe('Teams Suite', () => {
+    it('TS14868 Team Admin can use Next button to page through list in Manage Members', () => {
+        cy.loginAsNewUser().then(() => {
```

Comment:
Since we have all the users we need by default, we shouldn't need to create a new user for this test. Additionally, the below suggestion includes some cleanup around the chunking/batching.

```suggestion
        cy.apiLogin('user-1');

        // # Create new team and visit its URL
        cy.apiCreateTeam('test-team', 'Test Team').then((createResponse) => {
            const testTeam = createResponse.body;
            cy.visit(`/${testTeam.name}`);

            cy.apiGetUsersNotInTeam(testTeam.id).then((getResponse) => {
                const users = getResponse.body;

                const usersToAdd = users.slice(0, 59).map((user) => {
                    return {
                        team_id: testTeam.id,
                        user_id: user.id,
                    };
                });

                Cypress._.chunk(usersToAdd, 20).forEach((chunk) => {
                    cy.apiAddUsersToTeam(testTeam.id, chunk);
                });
            });
        });
```

This way, we would only have a team of 60 users, so you'll need to update your assertions below as well.

Reply:
Yes, applied these changes.